go/libraries/doltcore/sqle/dtables/conflicts_tables.go

```diff
@@ -43,22 +43,22 @@ type RootSetter interface {
 
 // NewConflictsTable returns a new ConflictsTableTable instance
 func NewConflictsTable(ctx *sql.Context, tblName string, root *doltdb.RootValue, rs RootSetter) (sql.Table, error) {
-	tbl, ok, err := root.GetTable(ctx, tblName)
-
+	var tbl *doltdb.Table
+	var ok bool
+	var err error
+	tbl, tblName, ok, err = root.GetTableInsensitive(ctx, tblName)
```

Comment:
Seems silly to declare all these vars just to reuse one variable here

Reply:
Same comment in all these files

Reply:
This `:=` syntax is not my favorite